.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -0,0 +1,47 @@
+### Description of change
+
+_Please provide description of change here_.
+
+### Pull Request check-list
+
+_Please make sure to review and check all of these items_:
+
+- [ ] Does `make test` pass after this change?
+- [ ] Is commit message formatted according to [CONTRIBUTING.md][0]
+
+### Affected core subsystem(s)
+
+- [ ] buffer
+- [ ] child_process
+- [ ] cluster
+- [ ] crypto
+- [ ] dgram
+- [ ] dns
+- [ ] doc
+- [ ] fs
+- [ ] http
+- [ ] https
+- [ ] net
+- [ ] tls
+- [ ] tty
+- [ ] vm
+- [ ] zlib
+- [ ] other _(please specify which)_
+
+### SemVer
+
+What [semver][1] change does this change require?
+
+- [ ] patch _(no new APIs, no breaking changes)_
+- [ ] minor _(new APIs, no breaking changes)_
+- [ ] major _(breaking changes, or just too dangerous to be minor/patch)_
+
+### LTS
+
+Should this patch be backported to:
```

Comment:
This sounds reasonable. I will change it once we will decide on general form of these files :)